.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+ci:
+  autofix_prs: false
```

Comment:
how do folks feel about switching this to true ?

Reply:
I think it's fine. The autofixes are almost always desirable, and having it enabled helps with third-party PRs where the original author might ignore the failing lint jobs in CI, and not commit the fixes themselves. By letting pre-commit take care of it in CI, third-party contributors don't need to deal with as many tools.